sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/Schema.java

```diff
@@ -181,6 +183,14 @@ public Builder addBooleanField(String name) {
       return this;
     }
 
+    public Builder addEnumerationField(String name, EnumerationType enumerationType) {
```

Comment:
Does it make sense to add these methods as first class add'ers to Schema? This feels like breaking consistency... as they are logical types.

Reply:
Originally I left them out. But then decided that these are fairly "basic" logical types (e.g. they exist in proto, avro, etc.) so it was worth adding this method as syntactic sugar. I don't feel very strongly about it either way though.